### ISSUE TITLE
Sequence Coordinates

### ISSUE THREAD

**Issue by almasaeed2010** [statonlab/hardwoods_site#428]
> for sequence coordinates, it says “This feature is not located on any sequence.“. I know this is a tripal thing, because we aren’t storing the chromosomes in chado, but we do actually know where these are located on a chromosome

**Comment by mestato** [comment]
We are going to start loading these from gff. Bradford did this for Fraxinus excelsior by creating empty "dummy" fasta files for the scaffolds with no sequences.

**Comment by mestato** [comment]
@cricha59 please add to minimum info list

**Comment by almasaeed2010** [comment]
@bradfordcondon when you get a chance, could you describe what needs to be done to make this happen? Thanks!

**Comment by bradfordcondon** [comment]
https://www.bradfordcondon.com/2018-03-28-loading_gff3_tripal/

do you need something more specific than this?

**Comment by almasaeed2010** [comment]
no I think  I can figure it out from here. Possibly right a script that the undergrads can simply use to extract the data.

**Comment by CaseyRichards92** [comment]
@almasaeed2010 did we ever figure out a way ahead with this?

**Comment by almasaeed2010** [comment]
I don't think so. It's a bit complicated honestly and might require that we change our way of loading data. Let's see what kind of data we want to extract from the GFF this Friday then take it from there.
